Client/Frontend/Browser/TabMetadataManager.swift

```diff
@@ -166,6 +166,7 @@ class TabMetadataManager {
     /// if the title isEmpty we don't record because title can be overriden
     /// - Parameters:
     ///   - url: Site URL from webview
```

Comment:
Url isn't a param there anymore it seems
```suggestion
```

Reply:
Yap url was removed